src/core/utils.js

```diff
@@ -61,3 +66,48 @@ export function getClientApp(userAgentString) {
 export function isValidClientApp(value, { _config = config } = {}) {
   return _config.get('validClientApplications').includes(value);
 }
+
+purify.addHook('afterSanitizeAttributes', (node) => {
+  // Set all elements owning target to target=_blank
+  // and add rel="noreferrer".
+  if ('target' in node) {
+    node.setAttribute('target', '_blank');
+    node.setAttribute('rel', 'noreferrer');
+  }
+});
+
+export function sanitizeHTML(text, allowTags = []) {
+  // TODO: Accept tags to allow and run through dom-purify.
+  return {
+    __html: purify.sanitize(text, { ALLOWED_TAGS: allowTags }),
+  };
+}
+
+// Convert new lines to HTML breaks.
+export function nl2br(text) {
+  return text.replace(/\n/g, '<br />');
```

Comment:
it looks like [clean_nl()](https://github.com/mozilla/addons-server/blob/master/src/olympia/amo/utils.py#L428) takes care of them.